### ISSUE TITLE
Segfault upon viewing issue from explorer

### ISSUE THREAD

**Issue by h0adp0re** [ankitpokhrel/jira-cli#199]
**Describe the bug**

Opening the issue explorer with the following command, then pressing `v` on an issue, brings up a segmentation violation.
```
❯ jira issue list -s"Open" -t~"Epic" --columns key,assignee,status
⠸ Fetching issues... panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x70 pc=0x1718567]

goroutine 27 [running]:
github.com/ankitpokhrel/jira-cli/internal/view.Issue.String({0x0, {0x0, 0x0, 0x0, {0x0, 0x0, 0x0}}})
        /Users/ankitpokhrel/go/src/github.com/ankitpokhrel/jira-cli/internal/view/issue.go:42 +0x67
github.com/ankitpokhrel/jira-cli/internal/view.IssueList.Render.func1.2({0x1787a60, 0x0})
        /Users/ankitpokhrel/go/src/github.com/ankitpokhrel/jira-cli/internal/view/issues.go:66 +0x77
github.com/ankitpokhrel/jira-cli/pkg/tui.(*Table).initTable.func2.1.1()
        /Users/ankitpokhrel/go/src/github.com/ankitpokhrel/jira-cli/pkg/tui/table.go:199 +0x25
github.com/rivo/tview.(*Application).Suspend(0xc0003ac460, 0xc0006b3f88)
        /Users/ankitpokhrel/go/src/github.com/ankitpokhrel/jira-cli/vendor/github.com/rivo/tview/application.go:519 +0xa9
github.com/ankitpokhrel/jira-cli/pkg/tui.(*Table).initTable.func2.1()
        /Users/ankitpokhrel/go/src/github.com/ankitpokhrel/jira-cli/pkg/tui/table.go:199 +0x157
created by github.com/ankitpokhrel/jira-cli/pkg/tui.(*Table).initTable.func2
        /Users/ankitpokhrel/go/src/github.com/ankitpokhrel/jira-cli/pkg/tui/table.go:192 +0x1a5
```
**Please provide following details**
1. JiraCLI Version:
    ```
    (Version="0.1.1", GitCommit="6663763b44a2e5d328b551ae7d2f1a37d1e232ae", GoVersion="go1.17.2", BuildDate="2021-11-03"", Compiler="gc", Platform="darwin/amd64")
    ```
2. Are you using Jira cloud or on-premise jira server? Also mention the version for on-premise installation.
    ```
    Jira Server v8.20.0
    ```
3. What operating system are you using? Also mention version.
    ```
    macOS 12.0.1
    ```
4. What terminal are you using? Also mention version.
    ```
    alacritty 0.9.0 (fed349a)
    ```

**To Reproduce**

1. `jira issue list -s"Open" -t~"Epic" --columns key,assignee,status`
2. Press `v` on an issue.
3. Witness the error

**Expected behavior**

Issue detail view opens.

**Comment by ankitpokhrel** [comment]
The problem is that the tui assumes `key` is always in second column while rendering. Since you changed the order of the key it panicked (which it shouldn't). I will see what I can do here. Thank you for reporting the issue.

This should work in your case.
```
jira issue list -s"Open" -t~"Epic" --columns assignee,key,status
```

**Comment by h0adp0re** [comment]
Hmm, that explains it, yes. However having anything other than `key` in the first column is impractical for me, since that's the sticky column. The issue key is the most important bit of info for me in this context.

I appreciate the speedy reaction and explanation though, thanks!

**Comment by ankitpokhrel** [comment]
@h0adp0re This is fixed with #202. I will make a new release in a few days. 

If you want to try the fix, you can [compile from source](https://github.com/ankitpokhrel/jira-cli#development) or if you are using homebrew you can build binary from `HEAD` (requires `make`, `git` and `go`)

```sh
brew uninstall jira-cli
brew install jira-cli --HEAD
```

**Comment by h0adp0re** [comment]
It works! Thank you!

**Comment by h0adp0re** [comment]
@ankitpokhrel I've now discovered that this fix creates another small issue. `^ + K` copies the second column to the clipboard. In my case it's issue type, which isn't useful.

**Comment by ankitpokhrel** [comment]
Crap! I will check it later today. Thank you for reporting!

**Comment by ankitpokhrel** [comment]
> @ankitpokhrel I've now discovered that this fix creates another small issue. `^ + K` copies the second column to the clipboard. In my case it's issue type, which isn't useful.

@h0adp0re this should be fixed in #204

**Comment by h0adp0re** [comment]
It is! Thank you again!
